app/dcr_monitor/main.go

```diff
@@ -3,28 +3,55 @@ package main
 import (
 	"context"
 	"fmt"
+	"net/http"
+	"time"
 
 	"github.com/cloudwego/hertz/pkg/common/hlog"
+	"github.com/manatee-project/manatee/app/dcr_api/biz/dal/db"
 	"github.com/manatee-project/manatee/app/dcr_monitor/client"
 	"github.com/manatee-project/manatee/app/dcr_monitor/monitor"
 	"github.com/manatee-project/manatee/pkg/config"
 )
 
 func main() {
+	// Set up HTTP server for liveness probe
+	http.HandleFunc("/healthz", healthHandler)
```

Comment:
Why do we need this part?

Reply:
actually we do not need it. I didn't remove it after debugging.